### PR TITLE
wait for controller POD ready in Github Action

### PR DESCRIPTION
**Pull request by freeznet**
### Motivation

Fix failed action https://github.com/streamnative/function-mesh/runs/7015101062?check_suite_focus=true

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Review by tpiperatgod — APPROVED** [review]
lgtm

File: .github/workflows/test-integration-kind-samples.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ jobs:
           make deploy
           kubectl get crds
           kubectl wait --for condition=available --timeout=360s deployment/function-mesh-controller-manager
```
Comment (tpiperatgod) [inline .github/workflows/test-integration-kind-samples.yml:74]: Is it only needed to keep "kubectl wait --for condition=ready" here?

Comment (freeznet) [reply .github/workflows/test-integration-kind-samples.yml]: thanks, addressed in the latest commit

**Review by jiangpengcheng — APPROVED** [review]
lgtm

**Pull request merged**